frontend/src/scenes/insights/Insights.tsx

```diff
@@ -142,20 +187,70 @@ export function Insights() {
                         ),
                     }}
                 >
-                    <TabPane tab={<span data-attr="insight-trends-tab">Trends</span>} key={ViewType.TRENDS} />
-                    <TabPane tab={<span data-attr="insight-funnels-tab">Funnels</span>} key={ViewType.FUNNELS} />
-                    <TabPane tab={<span data-attr="insight-sessions-tab">Sessions</span>} key={ViewType.SESSIONS} />
-                    <TabPane tab={<span data-attr="insight-retention-tab">Retention</span>} key={ViewType.RETENTION} />
-                    <TabPane tab={<span data-attr="insight-path-tab">User Paths</span>} key={ViewType.PATHS} />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-trends-tab">
+                                Trends
+                                <InsightHotkey hotkey="t" />
+                            </span>
+                        }
+                        key={ViewType.TRENDS}
+                    />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-funnels-tab">
+                                Funnels
+                                <InsightHotkey hotkey="f" />
+                            </span>
+                        }
+                        key={ViewType.FUNNELS}
+                    />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-sessions-tab">
+                                Sessions
+                                <InsightHotkey hotkey="s" />
+                            </span>
+                        }
+                        key={ViewType.SESSIONS}
+                    />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-retention-tab">
+                                Retention
+                                <InsightHotkey hotkey="r" />
+                            </span>
+                        }
+                        key={ViewType.RETENTION}
+                    />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-path-tab">
+                                User Paths
+                                <InsightHotkey hotkey="p" />
+                            </span>
```

Comment:
Funny case: when I press "P", I can't use any other hotkey as a input on that page has autofocus :)

Reply:
Yeah discovered that too, not 100% sure about the best way forward, seems like having that open could be pretty useful too. Maybe I'll enable a `Esc` press there so that you can get back to navigating with just your keyboard

Reply:
I'd remove the autofocus on all pages. This really hurts mobile navigation in general.

Reply:
I think the best UX here will be to keep autofocus on desktop and remove on mobile, but definitely out of the scope for this PR. For now I think this is a good argument, will remove autofocus from here.